scripts/web/install.sh

```
@@ -261,11 +282,67 @@ install_binary() {
     chmod +x "$INSTALL_DIR/$BINARY_NAME"
 }
 
+# Setup shell completions
+setup_shell_completions() {
+    local shell_type="$(detect_shell_type)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)